gamestonk_terminal/common/behavioural_analysis/sentimentinvestor_view.py

```diff
@@ -1,219 +1,71 @@
-    metric : str
-        Metric to get data for. Either 'sentiment', 'AHI', 'RHI', 'SGP'
+    Returns
+    -------
     """
-    if not sentipy.supported(ticker):
-        print("This stock is not supported by the SentimentInvestor API.")
-        return
-
-    df = sentimentinvestor_model.get_historical(ticker, metric)
-
-    if df.empty:
-        print("The dataset is empty, something must have gone wrong")
-        return
-
-    _customise_plot()
-
-    # use seaborn to lineplot
-    ax = sns.lineplot(
-        data=df,
-        x="date",
-        y=metric,
-        legend=False,
-        label=[metric],
-    )
 
-    # always show zero on the y-axis
-    plt.ylim(bottom=0)
 
-    # set the x-axis date formatting
-    ax.xaxis.set_major_formatter(mdates.DateFormatter("%x"))
+    df = sentimentinvestor_model.get_historical(ticker, start, end, limit)
 
-    # scale the plot appropriately
-    plt.gcf().set_size_inches(plot_autoscale())
-    plt.gcf().set_dpi(PLOT_DPI)
-    plt.gcf().autofmt_xdate()
-
-    # fill below the line
-    plt.fill_between(df.date, df[metric], alpha=0.3)
-
-    # add title e.g. AAPL sentiment since 22/07/21
-    plt.title(f"{ticker} {metric} since {min(df.date).strftime('%x')}")
-    if gtff.USE_ION:
-        plt.ion()
-    plt.show()
-
-    ###
-
-    boundary = _Boundary(0, max(df[metric].max(), 2 * df[metric].mean()))
-
-    # average for each day
-    aggregated = df.resample("D", on="date").mean()
-
-    # reverse the ordering if requested
-    aggregated.sort_values(
-        metric if sort_param == "value" else sort_param,
-        axis=0,
-        ascending=sort_dir == "asc",
-        inplace=True,
-    )
-
-    # format the date according to user's locale
-    aggregated.index = aggregated.index.strftime("%x")
-
-    # apply coloring to every value
-    aggregated[metric] = [
-        boundary.categorise(value)[0] + str(value) + Style.RESET_ALL
-        for value in aggregated[metric]
-    ]
-
-    print(
-        tabulate.tabulate(
-            aggregated,
-            headers=["Day", f"average {metric}"],
-            tablefmt="psql",
-            floatfmt=".3f",
-        ),
-        "\n",
-    )
+    if df.empty:
+        print("Error in Sentiment Investor request")
+    else:
+        _, ax1 = plt.subplots(figsize=(25, 7))
+        ax1.plot(df.index, df["RHI"], c="k")
+        ax2 = ax1.twinx()
+
+        ax1.grid()
+        ax2.plot(df.index, df["AHI"], c="orange")
+
+        ax1.set_ylabel(f"RHI")
+        ax1.set_xlabel("Time")
+        ax1.set_title(
+            f"Hourly-level data of RHI and AHI"
+        )
+        ax1.set_xlim(df.index[0], df.index[-1])
+        ax2.set_ylabel(f"AHI")
+
+        interval_locator = mdates.HourLocator(interval=6)
+        plt.gca().xaxis.set_major_locator(interval_locator)
+        plt.gca().xaxis.set_major_formatter(mdates.DateFormatter("%Y-%m-%d %H:%M"))
+        plt.gcf().autofmt_xdate()
+
+        if gtff.USE_ION:
+            plt.ion()
+
+        plt.show()
+
```

Comment:
So here you can add a raw argument, then `if raw:  print data` as we do elsewhere.